### PR TITLE
Add French translation

### PR DESCRIPTION
**Pull request by Ribesg**
This is a first version, some things may need to be tweaked at a later point

### CODE REVIEW

File: src/main/resources/translations/Messages_fr.properties [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+alias_friends = fs
+alias_setowner = so
+alias_transferblocks = tb;transfer;transferb
+alias_transferplayer = transferp
+alias_unfriend = ufr;uf
+
+### MAIN PLUGIN MESSAGES ###
+
+plugin_disabled_config = Le plugin est désactivé dans la configuration. Modifiez-là ou il ne fera rien !
+plugin_loaded_players = Chargé ! %s joueurs trouvés.
```
Comment (gdude2002) [inline src/main/resources/translations/Messages_fr.properties]: I don't 100% understand French semantics - Should there be a space before that `!`?

**Comment by gdude2002** [comment]
This looks pretty good to me; would some other command aliases make sense though? Part of the point of offering that is so that we can provide localised commands as well, eg, `amis` for `friends`.

Overall, though, this looks pretty great - If you're satisfied with it and have had a look at aliases, it should be fine to merge.

**Comment by bendem** [comment]
Nice translation @Ribesg, got nothing to say :+1:

**Comment by Ribesg** [comment]
I didn't know what to do for aliases. We're not used to use french commands here. It would feel weird for me to use french commands, but I don't know for your general users. Maybe we could add some aliases for simple user commands.

**Comment by gdude2002** [comment]
Aliases would generally just be the actual commands, but translated. So, `/friends` might become `/amis` or `/amies`, with `/as` as a shortened version, perhaps?

The English long commands would still be there, but I figured that we shouldn't expect people who don't know English to use meaningless English words for command names.

**Comment by Ribesg** [comment]
If anyone knows how you could translate "unfriend" just tell me :P

**Comment by gdude2002** [comment]
Hmm.. I guess I didn't think about French grammar in that respect.

I was going to post a link, but my PC randomly took a screenshot instead. Unfortunately it was the wrong screenshot, so, [how's this](http://dictionary.reverso.net/english-french/unfriend)?

**Comment by Ribesg** [comment]
Yeah but I mean, it's really long. I don't see any translation short enough.

**Comment by gdude2002** [comment]
Yeah, fair point. I guess there isn't something to substitute in?

**Comment by Ribesg** [comment]
We could just prefix the "positive" alias (ami/a) with "non/n" to get "nonami/na"...

Well I think na works but nonami feels strange xD

**Comment by gdude2002** [comment]
NOTFRIEND. lol.

Well, it could work. It seems intuitive enough.

**Comment by Ribesg** [comment]
Ready to be merged!

**Comment by gdude2002** [comment]
Lol, nice commit message. Thanks a ton for your work \o/

**Pull request merged**